assets/src/edit-story/components/canvas/canvasUploadDropTarget.js

```diff
@@ -47,13 +51,7 @@ function CanvasUploadDropTarget({ children }) {
       <UploadDropTargetOverlay>
         <CanvasLayer>
           <PageArea>
-            <UploadDropTargetMessage
-              id={MESSAGE_ID}
-              message={__(
-                'Upload to media library and add to the page.',
-                'web-stories'
-              )}
-            />
+            <UploadDropTargetMessage id={MESSAGE_ID} message={message} />
```

Comment:
Hmm. Why would this have any value?

Reply:
Interesting, I tried this again and now it has no effect. Not sure why I flagged this earlier, but I don't think we need this.